test/split_single_dyn_dim_test.cpp

```diff
@@ -157,64 +157,4 @@ TEST_CASE(multiple_outputs)
     EXPECT(p0 == p1);
 }
 
-TEST_CASE(broadcast_match)
```

Comment:
If i'm not mistaken your new pass should handle this case then and remove the need for this test?

Reply:
Yep